app/templating/view_context.py

```diff
@@ -5,10 +5,10 @@
 from app.helpers.form_helper import get_form_for_location
 from app.templating.summary_context import build_summary_rendering_context
 from app.templating.template_renderer import renderer
+from app.templating.summary.question import Question
```

Comment:
This is being imported so that the `get_title_from_titles` function can be called. I think that implies that the function is in the wrong place because it isn't specific to the summary.

Consider adding a `utils` module to the `templating` package for now and add the function there instead.

Reply:
Your spot on. Adding it to view context caused circular dependencies . Moving out to a utils file